fighthealthinsurance/workflows/send_fax.py

```
@@ -0,0 +1,84 @@
+with workflow.unsafe.imports_passed_through():
+    from fighthealthinsurance.activities import fax as fax_activities
+
+# How long to wait before sending when ``delay_send`` is set. Mirrors the old
+# fax-polling actor's "older than 1 hour" threshold.
+DELAYED_SEND_WAIT = timedelta(hours=1)
+
+
+@workflow.defn
+class SendFaxWorkflow:
+    @workflow.run
+    async def run(self, input: SendFaxInput) -> bool:
```

Ruff (0.15.17)

[error] Function argument `input` is shadowing a Python builtin

(A002)